realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsUnmanagedTests.java

```diff
@@ -53,7 +53,7 @@ public void tearDown() {
         }
     }
 
-    // When unmanaged, an object's backlinks fields have their initialized value (probably null).
```

Comment:
Is it a requirement that the initialized value be null, now?  It certainly is not forced by any code that I wrote...  If it is a requirement, I'd like to see a test that demonstrates that we enforce it.  If it is not a requirement, then I would prefer that we not claim that it is.